### DISCUSSION TITLE
PAG Viewer ARM 支持

### DISCUSSION THREAD

**Discussion by Nobit93** [Tencent/libpag#400] [Ideas] [answered]
请问开发团队有没有计划让 AE 导出插件可以支持 arm 架构？
AE 已经原生支持了 Apple Silicon, 但每次要导出的时候得切换回 rosetta, 性能直线下降。
多谢。

**Comment by kevingpqi123** [comment, marked as answer]
计划是下个版本，预计这个月可以支持

**Reply by Nobit93** [reply]
好像还没有看到更新？

**Reply by kevingpqi123** [reply]
由于人力问题，目前这部分有所 delay，适配过程中也遇到一些问题，适配 OK 后会第一时间同步给大家

**Reply by domchen** [reply]
4.2版本已经正式支持 M1 设备了。
